crates/matrix-sdk-ui/src/notification_client.rs

```diff
@@ -40,6 +40,8 @@ impl NotificationClient {
     ) -> Result<Option<NotificationItem>, Error> {
         let Some(room) = self.client.get_room(&room_id) else { return Err(Error::UnknownRoom) };
 
+        let mut retry_decryption = self.retry_decryption;
```

Comment:
Typo in commit message: `iloop`

Reply:
Considering that the history isn't tidy anymore :cry: I'll squash-merge this PR.